server/src/test/java/org/eclipse/lsp/cobol/service/utils/TestThreadPoolExecutorTest.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2020 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+package org.eclipse.lsp.cobol.service.utils;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * Test {@link TestThreadPoolExecutor}
+ */
+class TestThreadPoolExecutorTest {
+  private static final int THE_MAGIC_NUMBER = 42;
```

Comment:
isn't it 41 ;)

Reply:
I've taken it from google calculator =)
![image](https://user-images.githubusercontent.com/60617592/111658829-ff59ea80-880c-11eb-8806-6e58aac0f1fe.png)